sites/partners/src/components/listings/PaperListingForm/sections/PreferencesAndPrograms.tsx

```diff
@@ -71,25 +76,26 @@ const ProgramsAndPreferences = ({
 
   return (
     <>
-      <SelectAndOrder
-        addText={t("listings.addPreference")}
-        drawerTitle={t("listings.addPreferences")}
-        drawerSubtitle={
-          process.env.showLottery && listing?.lotteryOptIn
-            ? t("listings.lotteryPreferenceSubtitle")
-            : null
-        }
-        editText={t("listings.editPreferences")}
-        listingData={preferences || []}
-        setListingData={setPreferences}
-        subtitle={t("listings.sections.housingPreferencesSubtext")}
-        title={t("listings.sections.housingPreferencesTitle")}
-        drawerButtonText={t("listings.selectPreferences")}
-        dataFetcher={useJurisdictionalMultiselectQuestionList}
-        formKey={"preference"}
-        applicationSection={MultiselectQuestionsApplicationSectionEnum.preferences}
-      />
-
+      {enablePreferences && (
+        <SelectAndOrder
```

Comment:
Issue: This is an edge case but if you do the following preferences aren't properly deleted.

1. Create a listing with a jurisdiction that has preferences enabled and attach at least one preference to it
2. Save the listing
3. Edit the listing and change to a jurisdiction that has preferences disabled. Save the listing.
4. The frontend still sends the request to the backend the old preferences under `listingMultiselectQuestions` which are then saved